scripts/thaliPostInstall.js

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env node
+
+//ThaliCordova_Plugin post installation script
+
+var fs = require('fs');
+var exec = require('child_process').execSync;
+var path = require('path');
+
+var rootdir = process.argv[2];
+//Dummy file to prevent the execution of this script when the developer adds other Cordova plugins
+var dummyFile = path.join(rootdir, '/../plugins/org.thaliproject.p2p/dummy.js');
+
+//Replace JXcoreExtension.java
+function replaceJXCoreExtension(){
+    var sourceFile =  path.join(rootdir + '/../plugins/org.thaliproject.p2p/src/android/java/io/jxcore/node/JXcoreExtension.java');
+    var targetFile = path.join(rootdir + '/../platforms/android/src/io/jxcore/node/JXcoreExtension.java');
+    fs.writeFileSync(targetFile, fs.readFileSync(sourceFile));
+}
+
+//Install thali node modules to global
+//once we have the thali module available in npm repo, this is step is not required
+function installThaliModules(){
```

Comment:
Do we need this function anymore?

If I understand things correctly then we begin an install by going to www/jxcore, requiring a package.json that contains thali as an entry and then running "jx --install" which will the download all the JS files from NPM and run the post install script which will run the cordova plugin which will then run this script. So by the time we get to this script all the thali JS files have already been downloaded? So what's the point of this command?